shared/src/Datadog.Trace.ClrProfiler.Native/instrumented_assembly_generator/instrumented_assembly_generator_helper.h

```diff
@@ -139,6 +141,13 @@ inline bool IsInstrumentedAssemblyGeneratorEnabled()
     return false;
 }
 
+inline bool IsCopyingOriginalsModulesEnabled()
+{
+    const auto isCopyingOriginalsModulesEnabled =
+        shared::GetEnvironmentValue(cfg_copying_originals_modules_env);
+    return isCopyingOriginalsModulesEnabled == WStr("1") || isCopyingOriginalsModulesEnabled == WStr("true");
```

Comment:
I think we should avoid duplicating this logic and instead reuse the TryParseBooleanEnvironmentValue method.

Reply:
Fix in d6e34e581c70cc2271806c12db2c3d4f2caf78f1